scripts/release.sh

```diff
@@ -55,9 +56,15 @@ cp -f   scripts/create_cita_config.py      target/install/scripts/
 cp -rf scripts/txtool                      target/install/scripts/
 
 # 6) docker env
-cp -f scripts/env.sh                       target/install/
-cp -f scripts/daemon.sh                    target/install/
+cp -f  env.sh                              target/install/bin/cita-env
+cp -f  scripts/cita_config.sh              target/install/bin/cita-config
 
 # 7) amend info of system contract
 cp -f scripts/amend_system_contracts.sh    target/install/scripts/
 cp -f scripts/amend_system_contracts.py    target/install/scripts/
+
+# 8) delete building container
```

Comment:
Why delete the container at this script?

Reply:
The build container is just for build, developers will not need it when building is completed.

Reply:
ok. i misunderstand it